gui/default/syncthing/core/syncthingController.js

```diff
@@ -1610,6 +1610,10 @@ angular.module('syncthing.core')
             return $scope.currentDevice._editing == 'existing';
         }
 
+        $scope.editingDeviceNew = function() {
+            return $scope.currentDevice._editing == 'new';
```

Comment:
A comment stating that "new-pending" was intentionally left out here would be good.

Reply:
I've now added a helper function for `pending` too. It's not used at the moment though. I'm not sure if it's better to still have it or to replace it with a comment instead.

Reply:
Well, adding a pending device **is** adding a new device.  Could you please just apply the comment as given?

* `new-pending` instead of `pending`.
* No extra function that is never called.
* Comment in `editingDeviceNew()` that it intentionally consideres only `new`, but not `new-pending`.

Reply:
Done.

Reply:
The comment was not supposed to be about a "missing" function. Suggestion:

    // The "new-pending" value is intentionally disregarded here.

Please add **within** the function, as I wrote in my last comment.

Reply:
Done.